docs/en/observability/monitor-uptime-synthetics.asciidoc

```diff
@@ -25,8 +25,9 @@ In the {uptime-app}, you can monitor the status of network endpoints using the f
 | *HTTP monitor* | Monitor your website. The HTTP monitor checks to make sure specific endpoints return the correct
 status code and display the correct text.
 
+// lint ignore v4 v6
 | *ICMP monitor* | Check the availability of your hosts. The ICMP monitor uses ICMP (v4 and v6) Echo
-Requests to check the network reachability of the hosts you are pinging. This will tell you whether the
+Requests to check the ability to reach the network of the hosts you are pinging. This will tell you whether the
```

Comment:
I would verify this change with a developer. It looks like "network reachability" is an industry term.